include/mega/utils.h

```diff
@@ -362,6 +362,9 @@ std::string webdavnameescape(const std::string &value);
 
 void tolower_string(std::string& str);
 
+#ifdef __APPLE__
+int macOSmajorVersion();
```

Comment:
I think it's easier to read this way:
```suggestion
    int macOSmajorVersion();
```

Reply:
Other ifdefs within same file does not apply padding, so that's the reason to keep in the same way.